apps/api/src/test-data/utilities/coscrad-data-example.ts

```diff
@@ -35,7 +35,7 @@ export const buildTestInstance = <T = unknown>(
         throw new InternalError(
             `In order to build a test instance of: ${
                 Object.getPrototypeOf(target).name
-            }, it must have a static factory method called "toDto"`
+            }, it must have a static factory method called "fromDto"`
```

Comment:
Finally! I'm not sure what branch I fixed this on, but it still hasn't been merged so let's go ahead and get this in to avoid future confusion.